test/unit/mdc-list/foundation.test.js

```diff
@@ -996,3 +996,35 @@ test('#getSelectedIndex should be in-sync with setter method for multi-select ch
   foundation.setSelectedIndex([0, 2, 3]);
   assert.deepEqual([0, 2, 3], foundation.getSelectedIndex());
 });
+
+test('#setEnabled should remove class name mdc-list-item--disabled', () => {
```

Comment:
Can you merge this test with `#setEnabled should set aria-disabled to false when enabled is true`.

Same for below test. I think we can add just 2 unit tests that checks for class name and attribute changes at the same time.

Reply:
Done!